lang/en-US.json

```diff
@@ -21,34 +21,21 @@
     "amp_add_torrent": "&Add torrent",
     "amp_exit": "E&xit",
     "amp_view": "&View",
-    "amp_extensions": "&Extensions",
     "amp_preferences": "&Preferences",
     "amp_help": "&Help",
-    "amp_check_for_update": "&Check for update",
```

Comment:
I think this is used in the Updater plugin (src/plugins/updater/updater.cpp)

Reply:
Yes, the regex used missed this. Will take care.